py/client/pydeephaven/_table_ops.py

```diff
@@ -244,27 +278,34 @@ def get_stub_func(cls, table_service_stub: table_pb2_grpc.TableServiceStub) -> A
 
     def make_grpc_request(self, result_id, source_id) -> Any:
```

Comment:
type hints?

.... there are so many of these.  I'm not going to comment more on them, but they are all over this file.